public/index.js

```diff
@@ -5,6 +5,7 @@ Vue.createApp({
 		idCode: '',
 		isLoading: false,
 		showMobileInstructions: false,
+		timer: ''
```

Comment:
`setTimeout` and `setInterval` both return a number. It makes sense to initialize this as `null` imho.

Reply:
Fixed